### PR TITLE
Improve doc for filters

### PR DESCRIPTION
**Pull request by ger-benjamin**
Bound to https://github.com/camptocamp/ngeo/pull/3650

### CODE REVIEW

File: doc/administrator/mapfile.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ a template::
 
     TEMPLATE "fooOnlyForWFSGetFeature"
 
-This is a fake template, but this is required.
+This is a fake template, but this is required (for querier, as for filter !).
```
Comment (jwkaltz) [inline doc/administrator/mapfile.rst]: "and for filter"
or
"as well as for filter"

File: doc/administrator/mapfile.rst [hunk 2]
```diff
@@ -83,8 +83,11 @@ example::
 
 ``gml_types``
 
-  Always set this to ``auto``. This means that the layer's field type
-  information is obtained from the input feature driver (OGC).
+  Set this to ``auto``. This means that the layer's field type information is obtained from
+  the input feature driver (OGC). If the type is not well interpreted (notably if you have a
+  wrong operator within your filter), you can define manually your feature type
+  with ``"gml_[feature]_type" "[type]"`` where type are one of
```
Comment (jwkaltz) [inline doc/administrator/mapfile.rst]: where type is one of

File: doc/integrator/filter.rst [hunk 3]
```diff
@@ -3,8 +3,19 @@
 Filter (Querier)
 ================
 
+Available attributes and operators in filters
+---------------------------------------------
+
+All attributes defined as "exported" in the layer of you map server will be automatically
```
Comment (jwkaltz) [inline doc/integrator/filter.rst]: of your  mapserver

File: doc/integrator/filter.rst [hunk 4]
```diff
@@ -3,8 +3,19 @@
...
+available as filterable attribute. If the type, and so the operator on the attribute, is not
+adapted, you should adapt it manually in your layer definition.
```
Comment (jwkaltz) [inline doc/integrator/filter.rst]: suggestion:
"If the type, and so the operator on the attribute, is not adequate for filtering, you should adapt the type in your layer definition."

File: doc/integrator/filter.rst [hunk 5]
```diff
@@ -3,8 +3,19 @@
...
+See `administrator_mapfile` (WFS GetFeature) for more informations (MapServer only).
```
Comment (jwkaltz) [inline doc/integrator/filter.rst]: for more information

**Comment by ger-benjamin** [comment]
Thanks to you

File: doc/integrator/filter.rst (hunk 3 above)
Comment (ger-benjamin) [reply doc/integrator/filter.rst]: That can be qgis, arcgis... a server of maps

File: doc/administrator/mapfile.rst [hunk 6]
```diff
@@ -83,8 +83,11 @@ example::
 
 ``gml_types``
 
-  Always set this to ``auto``. This means that the layer's field type
-  information is obtained from the input feature driver (OGC).
+  Set this to ``auto``. This means that the layer's field type information is obtained from
+  the input feature driver (OGC). If the type is not well interpreted (notably if you have a
+  wrong operator within your filter), you can define manually your feature type
+  with ``"gml_[feature]_type" "[type]"`` where type is one of
```
Comment (sbrunner) [inline doc/administrator/mapfile.rst]: with ``"gml_<attribute>_type" "<type>"`` where `<type>` is one of

File: doc/administrator/mapfile.rst [hunk 7]
```diff
@@ -83,8 +83,11 @@ example::
...
+  these: ``Integer|Long|Real|Character|Date|Boolean``.
```
Comment (sbrunner) [inline doc/administrator/mapfile.rst]: less programmatic =>
```
``Integer``, ``Long``, ``Real``, ``Character``, ``Date``, ``Boolean``
```

File: doc/administrator/mapfile.rst (hunk 6 above)
Comment (ger-benjamin) [reply doc/administrator/mapfile.rst]: I've taken the same notation as MapServer, and as above in the same page. I'll adapt the whole page

File: doc/integrator/filter.rst (hunk 3 above)
Comment (jwkaltz) [reply doc/integrator/filter.rst]: yes, but it is "of your map server" (not "of you map server")

File: doc/administrator/mapfile.rst (hunk 6 above)
Comment (sbrunner) [reply doc/administrator/mapfile.rst]: Thanks :-)

File: doc/integrator/filter.rst (hunk 3 above)
Comment (ger-benjamin) [reply doc/integrator/filter.rst]: ah , ha, yes indeed

File: doc/administrator/mapfile.rst [hunk 8]
```diff
@@ -65,7 +65,7 @@ a template::
 
     TEMPLATE "fooOnlyForWFSGetFeature"
 
-This is a fake template, but this is required.
+This is a fake template, but this is required (for querier and for filter !).
```
Comment (sbrunner) [inline doc/administrator/mapfile.rst]: No space before `!` in English...

**Pull request merged**